xbmc/music/CueInfoLoader.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+/*
+ *      Copyright (C) 2005-2013 Team XBMC
```

Comment:
Might as well. We don't have a real reason for it after some online searching as we have all code online with date of contributions